test/functional/feature_pruning.py

```diff
@@ -34,15 +68,15 @@ def set_test_params(self):
 
         # Create nodes 0 and 1 to mine.
         # Create node 2 to test pruning.
-        self.full_node_default_args = ["-maxreceivebuffer=20000", "-checkblocks=5", "-limitdescendantcount=100", "-limitdescendantsize=5000", "-limitancestorcount=100", "-limitancestorsize=5000" ]
+        self.full_node_default_args = ["-dev", "-maxreceivebuffer=20000", "-checkblocks=5", "-limitdescendantcount=100", "-limitdescendantsize=5000", "-limitancestorcount=100", "-limitancestorsize=5000" ]
```

Comment:
Why `-dev` option necessary?

Reply:
No it's not necessary, all functional test nodes run in 'dev' mode. It's another attempt to run this test.  none of these changes make any difference to this test.